tests/test-cases/Group6-VIC-Machine/6-08-Create-Compute.robot

```diff
@@ -0,0 +1,83 @@
+    Log To Console  \nRunning vic-machine create
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+    Set Test VCH Name
+
+    Log To Console  \nInstalling VCH to test server...
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL} --compute-resource=%{TEST_RESOURCE} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD} ${vicmachinetls}
+    Should Contain  ${output}  Installer completed successfully
+    Log To Console  Installer completed successfully: ${vch-name}...
+
+    Run Regression Tests
+    Cleanup VIC Appliance On Test Server
+
+Compute resources - Default resource pool
+    Log To Console  \nRunning vic-machine create
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+    Set Test VCH Name
+
+    Log To Console  \nInstalling VCH to test server...
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD} ${vicmachinetls}
+    Should Contain  ${output}  Installer completed successfully
+    Log To Console  Installer completed successfully: ${vch-name}...
+
+    Run Regression Tests
+    Cleanup VIC Appliance On Test Server
+
+Compute resources - Non-Default resource pool
+    Log To Console  \nRunning vic-machine create
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+    Set Test VCH Name
+    Log To Console  Creating resource pool /ha-datacenter/host/localhost.localdomain/Resources/test1
+    Run  govc pool.create /ha-datacenter/host/localhost.localdomain/Resources/test1
+
+    Log To Console  \nInstalling VCH to test server...
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL} --compute-resource=%{TEST_RESOURCE}/test1 --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD} ${vicmachinetls}
+    Should Contain  ${output}  Installer completed successfully
+    Log To Console  Installer completed successfully: ${vch-name}...
+    ${out}=  Run  govc vm.info -json ${vch-name}
+    Log To Console  Checking the VM Info for resource pool
+    Should Contain  ${out}  POOL_PATH=/ha-datacenter/host/localhost.localdomain/Resources/test1
+    
+    Run Regression Tests
+    Cleanup VIC Appliance On Test Server
+
+    Run  govc pool.destroy /ha-datacenter/host/localhost.localdomain/Resources/test1
```

Comment:
Yes, adding govc pool.destroy "*/Resources/*" to Cleanup VIC Appliance On Test Server